sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -120,6 +124,19 @@ export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
         });
         newSettings.siteLogo = coverImageUrl;
       }
+      const handle = formik.values.siteTwitterHandle;
+      const website = formik.values.siteUrl;
+
+      formik.values.siteTwitterHandle = handle.split('/').pop() as string;
+      newSettings.siteTwitterHandle = handle.split('/').pop() as string;
+
+      if (handle.includes('@')) {
```

Comment:
if the if really necessary? replace will just not change the string if `@` is not found right?

Reply:
oh I see! I didn't know that was how replace was working, but that makes sense, thanks :)